File: A2/A2-027/A2-027.cpp
#include<iostream>
using namespace std;

int N, maxScore = 0, maxCount = 0;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);
    // Input number of scores
    cin >> N;
    // Find maximum score and count them
    while(N--)
    {
        int score;
        cin >> score;
        if(score == maxScore) { maxCount++; }
        else if(score > maxScore)
        {
            maxScore = score;
            maxCount = 1;
        }
    }
    // Output
    cout << maxScore << '\n' << maxCount;
    return 0;
}

File: A2/A2-027/A2-027.py
# Input number of scores
N = int(input())
# Find maximum score and count them
max_score = 0
max_count = 0
for _ in range(N):
    score = int(input())
    if(score == max_score):
        max_count += 1
    elif(score > max_score):
        max_score = score
        max_count = 1
# Output
print(max_score)
print(max_count)